go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/antihax/optional v1.0.0
 	github.com/banzaicloud/k8s-objectmatcher v1.4.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
-	github.com/erdrix/nigoapi v0.0.0-20211122092449-0fa36e567288
 	github.com/go-logr/logr v1.2.2
 	github.com/go-logr/zapr v1.2.3
 	github.com/imdario/mergo v0.3.12
 	github.com/jarcoal/httpmock v1.0.6
 	github.com/jetstack/cert-manager v1.7.2
+	github.com/konpyutaika/nigoapi v0.0.0-20220917224206-75aa546e554b
```

Comment:
Is there a reason to not used a released version of `nigoapi` here? For example `v0.5.0` or `v0.6.0`?

Reply:
I don't why but it doesn't work https://pkg.go.dev/github.com/konpyutaika/nigoapi

Reply:
I found why, the tag was `X.X.X` instead of `vX.X.X`.